tools/syz-declextract/run.go

```diff
@@ -72,15 +112,61 @@ func worker(outputs chan output, files chan string, binary, compilationDatabase
 
 		cmd := exec.Command(binary, "-p", compilationDatabase, file)
 		stdout, err := cmd.Output()
-		var stderr []byte
+		var stderr string
 		if err != nil {
 			var error *exec.ExitError
 			if errors.As(err, &error) {
-				stderr = error.Stderr
+				stderr = string(error.Stderr)
 			} else {
-				stderr = []byte(err.Error())
+				stderr = err.Error()
+			}
+		}
+		outputs <- output{string(stdout), stderr}
+	}
+}
+
+func renameSyscall(kernelDir string) func(string) string {
+	var rename = make(map[string]string)
+	var isValidSyscall = make(map[string]bool)
+	for _, arch := range targets.List[targets.Linux] {
+		filepath.Walk(kernelDir+arch.KernelHeaderArch, func(path string, info fs.FileInfo, err error) error {
+			if strings.HasSuffix(path, ".tbl") {
+				fi, err := os.Lstat(path)
+				if err != nil {
+					tool.Fail(err)
+				}
+				if fi.Mode()&fs.ModeSymlink != 0 { // Don't follow symlinks.
```

Comment:
The comment says what the code already says, so it does not add much value.

Explain _why_ we skip symlinks instead (it's unclear to me).

Reply:
Done!